### ISSUE TITLE
Crash after custom Badeline cutscene

### ISSUE THREAD

**Issue by swoolcock** [rhelmot/CelesteRandomizer#58]
Not sure whether this crash is from CelesteRandomizer or from the custom Badeline cutscene.
~~I assume~~ the cutscene is from "Badeline's Training" that I have installed: https://gamebanana.com/maps/207516
Edit: This is the one: https://youtu.be/wSVnlwFrcZk?t=778

I exited through the top of a room (can't remember which one) and landed on the hotel roof.  It immediately played one of the custom Badeline cutscenes, which I went through manually (didn't skip).  Once the cutscene was finished, the game froze for about 5-10 seconds (music still playing), then crashed.

I see this in the log, so assume it's the seed for the world:
`(08/03/2020 15:08:53) [Everest] [Verbose] [randomizer] new area randomizer/swlt4z_496073329`

Log: [log.txt](https://github.com/rhelmot/CelesteRandomizer/files/5014244/log.txt)

OS: macOS Mojave 10.14.6
Controller: Xbox Elite Series 2
Celeste: v.1.3.1.2 from itch.io
Everest: v.1.1847.0-azure-fcc35
CelesteRandomizer: v0.3.3
Speedrun Tool: v.2.1.11

Note that I've replaced the SDL library in Celeste with the latest from libsdl.org, as it adds support for my controller.

**Comment by rhelmot** [comment]
do you perhaps have an error_log.txt? your log.txt doesn't show the actual cause of the crash.

**Comment by swoolcock** [comment]
Hmm that's very odd, the log.txt I was looking at had a lot more info in it than the one I apparently uploaded, and now I can't find the original.  Here's the error_log.txt too.
[error_log.txt](https://github.com/rhelmot/CelesteRandomizer/files/5014696/error_log.txt)

**Comment by swoolcock** [comment]
I worked out the original settings I had to get that world:

```
Seed: swlt4z
Maps: Chapters 1, 2, 3, 4, 6, 7 A-sides only
Logic: Pathway
Length: Medium
Dashes: One
Difficulty: Normal
Repeat Rooms: Off
Enter Strange Holes: Off
Illuminate Path: Hub Rooms Only
Golden Strawberry: Off
```

I'll try to reproduce it and get a better log file.

**Comment by rhelmot** [comment]
This should be fixed in the next patch as per #61
